sshcode.go

```diff
@@ -451,7 +463,7 @@ func syncUserSettings(sshFlags string, host string, back bool) error {
 		return err
 	}
 
-	const remoteSettingsDir = "~/.local/share/code-server/User/"
+	var remoteSettingsDir = windowsVarFix(".local/share/code-server/User/")
```

Comment:
Why not consider something like `os.GetEnv("HOME")+"/.local/share/code-server/User/"` It's slightly more verbose yes but it's also more clear as to what is happening and _should_ remove the need for modifying the path at all (as it should work for all supported OS)?

Reply:
Nope. what that does is get the CLIENT home path, not the REMOTE home path, For local settings? that is a GREAT idea, will look into later for a future push.

Reply:
Thanks for the clarification.